<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlSettingsEdit</class>
 <widget class="QWidget" name="ControlSettingsEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Command</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="commandEdit"/>
   </item>
   <item>
    <widget class="QCheckBox" name="syncCallCheck">
     <property name="text">
      <string>call with 1 sec sync msg</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Layout</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="vLayBtn">
     <property name="text">
      <string>Vertical</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="hLayBtn">
     <property name="text">
      <string>Horizontal</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
